test/javascripts/lib/url-test.js.es6

```diff
@@ -25,3 +25,9 @@ test("isInternal on subfolder install", function() {
   not(DiscourseURL.isInternal("http://eviltrout.com/tophat"), "a url on the same host but on a different folder is not internal");
   ok(DiscourseURL.isInternal("http://eviltrout.com/forum/moustache"), "a url on the same host and on the same folder is internal");
 });
+
+test("userPath", assert => {
+  assert.equal(userPath(), '/u');
+  assert.equal(userPath('eviltrout'), '/u/eviltrout');
+  assert.equal(userPath('hp.json'), '/u/hp.json');
```

Comment:
Maybe a test case for subfolder paths as well since `Discourse.getUrl` is extracted into the function?